styles.css

```diff
@@ -1,65 +1,32 @@
-/* ******GLOBAL******** */
+/* ****** GENERAL STYLES ******** */
 html {
   background-color: #A9AAD2;
+  font-family: "Open Sans", sans-serif;
   height: 100%;
 }
 body {
-  border: 1px red solid;
-  font-family: "Open Sans", light;
   display: flex;
   align-items: stretch;
   justify-content: space-between;
   margin: 0;
+  font-size: 1em;
 }
-/* ****** ASIDE SECTION ******  */
-aside {
-  background-color: #1F1F3d;
-  color: #FFFFFF;
+/*  RIGHT SIDE SECTIONS  */
+section {
   display: flex;
-  align-items: flex-start;
+  width: 100%;
   flex-direction: column;
-  width: 18em;
-  height: 58em;
-}
-h2 {
-  padding-right: 1em;
-  padding-left: 1em;
-  font-weight: 500;
+  align-items: center;
+  background-color: #EAEAF4;
 }
```

Comment:
I would recommend giving this a class name and then assign styling by that class name